manifests/init.pp

```diff
@@ -48,7 +49,9 @@
   String $default_pad_text          = 'Welcome to etherpad!',
 
   # Users
-  Optional[Hash] $users = undef,
+  Optional[Hash] $users                  = undef,
+  Optional[String] $ep_local_admin_login = 'admin',
```

Comment:
can you use ```String[1]``` here and for the other new param?

Reply:
Fixed.